elements/dash-tabs/dash-tabs.e2etest.js

```diff
@@ -102,6 +102,7 @@ describe('dash-tabs', function() {
     async function() {
       const lastTab =
         await this.driver.findElement(By.css('[role=tab]:last-of-type'));
+      // FIXME: This line is missing an await
```

Comment:
oh yeah @surma this was a note I left to remind  you that I think this tabs test needs to await reading the attributes.